main/llvm4/APKBUILD

```diff
@@ -207,6 +209,15 @@ dev() {
 	_mv "$pkgdir"/$_prefix/bin/llvm-config $_prefix/bin/
 }
 
+lit() {
+	pkgdesc="A portable tool for executing LLVM and Clang style test suites"
+	pkgver=0.6.0
```

Comment:
This is the first time I see `pkgver` overridden in a subpackage. I’m not sure if it’s okay and what consequences it might have.